src/django/api/filters/submission.py

```diff
@@ -0,0 +1,28 @@
+from rest_framework.filters import BaseFilterBackend
+
+from ..models.role import Roles
+
+
+class UserSubmissionFilterBackend(BaseFilterBackend):
+    def filter_queryset(self, request, queryset, view):
+        if request.user.role == Roles.CONTRIBUTOR:
+            queryset.filter(boundary__utility__in=request.user.utilities)
+
+        if request.user.role == Roles.VALIDATOR:
+            queryset.filter(boundary__utility__state__in=request.user.states)
+
+        if request.user.role != Roles.ADMINISTRATOR:
+            # TODO: raise error
+            pass
+
+        return queryset
+
+
+class RequestSubmissionFilterBackend(BaseFilterBackend):
+    def filter_queryset(self, request, queryset, view):
+        requested_utilities = request.data.get('utilities')
+
+        if requested_utilities:
+            queryset.filter(utility__in=requested_utilities)
```

Comment:
.filter() doesn't operate in place.
```suggestion
            queryset = queryset.filter(utility__in=requested_utilities)
```